apps/design-land/src/app/checkbox/examples/generated_set/generated_set.component.ts

```diff
@@ -0,0 +1,25 @@
+import { Component, OnInit, ViewChild } from '@angular/core';
+import { FormArray, FormControl } from '@angular/forms';
+import { DaffCheckboxSetComponent } from '@daffodil/design';
+
+@Component({
+    selector: 'generated-set',
```

Comment:
Is an example for this needed? Generating a checkbox set from a list with an array isn't a UI thing that needs to be outlined? We already have the example of what a checkbox-set is.